apps/fluent-tester/src/FluentTester/TestComponents/Avatar/BasicAvatar.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { AvatarSize, AvatarColor, Avatar, AvatarPresence } from '@fluentui-react-native/experimental-avatar';
+import { Switch, View, Text, Picker, ColorValue } from 'react-native';
+import { satyaPhotoUrl, undefinedText } from './../PersonaCoin/styles';
+import { commonTestStyles as commonStyles } from '../Common/styles';
+import { useTheme } from '@fluentui-react-native/theme-types';
+import { AvatarTokens } from '@fluentui-react-native/experimental-avatar';
+
+type WithUndefined<T> = T | typeof undefinedText;
+
+const allSizes: WithUndefined<AvatarSize>[] = [
```

Comment:
If possible, we should pull these from the types that Avatar exports so we don't have to update them when we change them